modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1253,7 +1253,7 @@ protected function content_template() {
 					'aria-selected': 1 === tabCount ? 'true' : 'false',
 					'tabindex': 1 === tabCount ? '0' : '-1',
 					'aria-controls': 'e-n-tab-content-' + tabUid,
-					'aria-label': '<?php echo esc_html__( 'Open tab (Enter or Space) | Return (Escape)', 'elementor' ); ?>',
```

Comment:
This `aria-label` is not required as it duplicates the title from the inner text. You can remove the entire attribute.